app/viewmodels/ImplicitConversions.scala

```diff
@@ -22,8 +22,9 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.Key
 
 import scala.language.implicitConversions
 
-object ImplicitConversions {
+object ImplicitConversions extends ImplicitConversions
```

Comment:
Could potentially consider renaming this to `object implicits` too so we could `import viewmodels.implicits._`